src/main.rs

```diff
@@ -203,18 +233,18 @@ async fn main() -> anyhow::Result<()> {
             WorkerType::Reader => {
                 n_readers_finished += 1;
 
-                if let Ok((query_timings, total_time)) = res {
-                    let qps = query_timings.len() as f64 / total_time.as_secs_f64();
+                if let Ok(query_timings) = res {
+                    let reader_qps = query_timings.values().sum::<f64>() / query_timings.len() as f64;
```

Comment:
do you actually store the qps or avgqps here?

Reply:
avgqps, will rename vars